<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Explode Geometry</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Explode Options</string>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Geometry Type</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>241</width>
           <height>20</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="trimesh_option">
            <property name="text">
             <string>TriMesh</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mnmesh_option">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>MNMesh</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>291</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QCheckBox" name="add_shell_checkbox">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Add Shell Modifier    ----&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="offset_label">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="shell_offset">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="add_edit_mesh_checkbox">
           <property name="text">
            <string>Add Edit Mesh Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="collapse_modifier_stack_checkbox">
           <property name="text">
            <string>Collapse Modifier Stack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="center_pivot_checkbox">
           <property name="text">
            <string>Center Pivot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="delete_original_checkbox">
           <property name="text">
            <string>Delete Original</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="select_label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Selected Object(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selected_objects_string">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="explode_button">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Explode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
